doc/source/blog_posts/TRACER.ipynb

```diff
@@ -0,0 +1,2267 @@
+{
```

Comment:
<p>Can you read directly from Adam's repo? Pandas supports remote reading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ARM-DOE/pyart/pull/1296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Done!